compose.yml

```diff
@@ -5,7 +5,9 @@
 version: "3.9"
 services:
   assume_db:
-    image: timescale/timescaledb:latest-pg15
+    image: timescale/timescaledb-ha:pg15-all-oss
```

Comment:
this version is not working on my macbook. I am getting the following error. Can we make this postgis as optional dependency?
 ⠧ assume_db Pulling                                                                                                                                                                                                            
no matching manifest for linux/arm64/v8 in the manifest list entries

Reply:
I think we should switch to `timescaledev/timescaledb-ha:pg15-all-oss` which already has arm support.
See https://github.com/timescale/timescaledb-docker-ha/issues/259

Once the official timescale org image has it too, we can switch back.
Otherwise we could remove postgis from it, but I doubt that it will be pleasent to switch to manually afterwards :shrug: 

So I will switch to the dev image.